### ISSUE TITLE
The Web Vault does not display folders that have vault items assigned to them; It only displays empty folders in the My vault view.

### ISSUE THREAD

**Issue by SergeantConfused** [bitwarden/clients#4805]
### Steps To Reproduce

1. Log into our Bitwarden account via the Web Vault (https://vault.bitwarden.com/#/).
2. Make sure that (All vaults) is selected in the (FILTERS) pane on the left.
3. Create a folder named (Folder1).
4. Create a vault item named (Item in Folder1) and assign it to (Folder1).
5. Switch from (All vaults) to (My vault) in the (FILTERS) pane.

### Expected Result

To see Folder1 in the Folders list in the (FILTERS) pane.

### Actual Result

Folder1 is not displayed.

### Screenshots or Videos

N/A.

### Additional Context

This happens with any folder, including (No Folder); Essentially, it appears that the Web Vault shows only empty folders when (My vault) is selected. Also, this behaviour would persist even if the item is deleted; You'd need to permanently delete it from the Trash for the relevant folder to become empty and to appear in the Folder list.

### Operating System

Windows

### Operating System Version

Windows 11 Pro 22H2.

### Web Browser

Firefox

### Browser Version

110.0.

### Build Version

2023.2.0.

### Issue Tracking Info

- [X] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by cksapp** [comment]
> Wonder if someone or something's already fixed this. I just pulled the latest on the Master branch and haven't been able to reproduce running locally.
> 
> Also, I checked out the commit tagged "web-2023.2.0" and I was able to reproduce it.

Looks like [PR 4700](https://github.com/bitwarden/clients/pull/4700) was merged that solved this, just waiting on the next release for the fix to rollout.

**Comment by gsacavdm** [comment]
When is the next release?
I'm surprised it's taking this long to roll out.

**Comment by cksapp** [comment]
> When is the next release?
> I'm surprised it's taking this long to roll out.

Apparently soon according to the [planned maintenance notice](https://www.reddit.com/r/Bitwarden/comments/11xm9n8/maintenance_window_mar_21)

**Comment by Diesmo** [comment]
This is a duplicate of #4471 and can now be closed, as of release 2023.3.0
